.github/workflows/build.yml

```diff
@@ -514,173 +308,44 @@ jobs:
-                "owner": "liquibase",
-                "repo": "liquibase-pro",
-                "sha": currentBranchInfo.sha,
-                "state": thisStatus,
-                "context": "Stable Liquibase Core",
-                "target_url": currentBranchInfo.runHtmlUrl,
-                "description": "Latest Liquibase Core build"
-              });
-            }
-
-            if (runFunctionalTests) {
-              let testsBranch = await helper.findMatchingBranch("liquibase", "liquibase-pro-tests");
-
-              let notices = [];
-              if (context.payload.pull_request) {
-                console.log("Sending pull request information");
-                notices = ["Triggered by PR " + context.payload.pull_request.html_url];
-              }
-              console.log("Trigger functional tests");
-              let workflowDispatch = await github.rest.actions.createWorkflowDispatch({
-                "owner": "liquibase",
-                "repo": "liquibase-pro-tests",
-                "workflow_id": "test.yml",
-                "ref": testsBranch.name,
-                "inputs": {
-                  "liquibaseCommit": helper.getCurrentSha(),
-                  "runDescription": "Triggered by https://github.com/liquibase/liquibase/actions/runs/"+context.runId,
-                  "liquibaseBranch": helper.getCurrentBranch(),
-                  "runDetails": JSON.stringify({
-                    "liquibaseBranch": helper.getCurrentBranch(),
-                    "notices": notices
-                  })
-                }
-              });
-
-              let testHarnessBranch = await helper.findMatchingBranch("liquibase", "liquibase-test-harness");
-              if (testHarnessBranch.name == "main" || testHarnessBranch.name == "master") {
-                testHarnessBranch.name = "develop";
-              }
-            
-              console.log("Trigger Test Harness tests");
-              let testHarnessWorkflowDispatch = await github.rest.actions.createWorkflowDispatch({
-                "owner": "liquibase",
-                "repo": "liquibase-test-harness",
-                "workflow_id": "main.yml",
-                "ref": testHarnessBranch.name,
-                "inputs": {
-                  "liquibaseCommit": helper.getCurrentSha(),
-                  "runDescription": "Triggered by https://github.com/liquibase/liquibase/actions/runs/"+context.runId,
-                  "liquibaseBranch": helper.getCurrentBranch(),
-                  "liquibaseRepo": "liquibase/liquibase",
-                  "runDetails": JSON.stringify({
-                    "liquibaseBranch": helper.getCurrentBranch(),
-                    "notices": notices
-                  })
-                }
-              });
-            }
+  run-functional-tests:
+    needs: [ setup, build_publish_branch ]
+    uses: liquibase/liquibase/.github/workflows/run-functional-tests.yml@github-action-DAT-15775
```

Comment:
Should this line have `@github-action-DAT-15775` in it?

Reply:
this was taken care of